test-cases/gemini/gemini-1tb-10h.yaml

```diff
@@ -3,7 +3,7 @@ n_db_nodes: 3
 n_test_oracle_db_nodes: 1
 n_loaders: 1
 n_monitor_nodes: 1
-instance_type_db: 'i3.4xlarge'
+instance_type_db: 'i4i.4xlarge'
 instance_type_loader: 'm5.xlarge'
 
 user_prefix: 'gemini-1tb-10h'
```

Comment:
Line 27  (can't comment there) ---> need to change also instance_type_db_oracle for all gemini test-cases.

Reply:
line 27 `instance_type_db_oracle: 'i3.16xlarge'` -> `instance_type_db_oracle: 'i4i.16xlarge'`
Changed all appearances of `instance_type_db_oracle` to `i4i`.